translations/monero-core_zh-tw.ts

```diff
@@ -1079,235 +1079,242 @@ The following information will be deleted
         <source>Custom decorations</source>
-        <translation type="unfinished"></translation>
+        <translation>自訂裝飾</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsLayout.qml" line="67"/>
         <source>No Layout options exist yet in mobile mode.</source>
-        <translation type="unfinished"></translation>
+        <translation>行動裝置版本中尚無介面選項。</translation>
     </message>
 </context>
 <context>
     <name>SettingsLog</name>
     <message>
         <location filename="../pages/settings/SettingsLog.qml" line="68"/>
         <source>Log level</source>
-        <translation type="unfinished"></translation>
+        <translation>日誌層級</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsLog.qml" line="143"/>
         <source>Daemon log</source>
-        <translation type="unfinished">節點日誌</translation>
+        <translation>節點日誌</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsLog.qml" line="226"/>
         <source>command + enter (e.g &apos;help&apos; or &apos;status&apos;)</source>
-        <translation type="unfinished"></translation>
+        <translation>輸入指令後按下 enter 送出 (範例: help 或 status)</translation>
     </message>
 </context>
 <context>
     <name>SettingsNode</name>
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="114"/>
         <source>Local node</source>
-        <translation type="unfinished"></translation>
+        <translation>本機節點</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="132"/>
         <source>The blockchain is downloaded to your computer. Provides higher security and requires more local storage.</source>
-        <translation type="unfinished"></translation>
+        <translation>區塊鏈將會下載到你的電腦中，這帶來較高的安全性但需要較多的儲存空間。</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="213"/>
         <source>Remote node</source>
-        <translation type="unfinished">遠端節點</translation>
+        <translation>遠端節點</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="232"/>
         <source>Uses a third-party server to connect to the Monero network. Less secure, but easier on your computer.</source>
-        <translation type="unfinished"></translation>
+        <translation>使用第三方的遠端節點以連接到 Monero 網路，較不安全，但簡易使用。</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsNode.qml" line="277"/>
         <source>To find a remote node, type &apos;Monero remote node&apos; into your favorite search engine. Please ensure the node is run by a trusted third-party.</source>
-        <translation type="unfinished"></translation>
+        <translation>如果需要遠端節點，上網搜尋 「 Monero 遠端節點 」。請確認該節點是由可信任的第三方所架設。</translation>
```

Comment:
@erciccione Could you please check if the directly using `「` and `」` here is technically OK with encoding?

Reply:
It is :)

Reply:
There is a space between ```尋``` and ```「```.
I think it can be removed.